library/network/src/modules/NetworkInterfaces.rb

```diff
@@ -681,6 +683,20 @@ def ConcealSecrets(devs)
       deep_copy(out)
     end
 
+    # Get current sysconfig configured interfaces
+    #
+    # @param devregex [String] regex to filter by
+    # @return [Array] of ifcfg names
+    def get_devices(devregex = "[~]")
```

Comment:
You can improve it 👍 . It was one of my first cleanups and just use same match that was used with `Builtins.regexpmatch`

https://github.com/yast/yast-yast2/pull/433/files#diff-36f13a6d2bafc7ad7bcfa6f8a452e81dL704

PR: https://github.com/yast/yast-yast2/pull/433